scripts/aggregate-translations/aggregate-translations.js

```diff
@@ -17,11 +17,34 @@ const defaultSearchPatterns = baseDirectory => ([
 const customDirectories = (baseDirectory, directories) =>
   (directories.map(dir => path.resolve(baseDirectory, dir)));
 
+const isFile = filePath => (fse.existsSync(filePath) && !fse.lstatSync(filePath).isDirectory());
+
+const configFile = (configPath) => {
+  if (configPath) {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    return require(configPath);
+  }
+
+  const localPath = path.resolve(process.cwd(), 'terraI18n.config');
+  if (isFile(localPath)) {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    return require(localPath);
+  }
+  return {};
+};
+
+const defaults = (options = {}) => {
+  const config = configFile(options.configPath);
+  return {
+    baseDir: options.baseDir || config.baseDir || process.cwd(),
+    directories: options.directories || config.directories || [],
+    fileSystem: options.outputFileSystem || config.outputFileSystem || fse,
+    locales: options.locales || config.locales || supportedLocales,
```

Comment:
This should probably be changed to 
```diff
-locales: options.locales || config.locales || supportedLocales,
+locales: options.locales || config.locales || [],
```

`supportedLocales` fallback would be undefined. Its pulled in like 
```const supportedLocales = require('./i18nSupportedLocales');```

Reply:
The fallback is defined. Supported locals is in the aggregate translations folder.
https://github.com/cerner/terra-toolkit/blob/webpack4/scripts/aggregate-translations/i18nSupportedLocales.js